server/test/comments.js

```diff
@@ -0,0 +1,24 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import server from '../../server';
+
+chai.use(chaiHttp);
+chai.should();
+
+const token = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyaWQiOjIsImFkbWluIjp0cnVlLCJpYXQiOjE1NDgyMjIzMDJ9.jeI-rw4tS4lbj7m1PzXLf0i75D6CwRTmmq_ExiArOhY';
+
+describe('/Post comments/:questionid', () => {
```

Comment:
Instead of multiple Describes, use one describe and multiple ITs. A file is expected to have one Describe block